apps/prairielearn/assets/scripts/instructorAssessmentStatisticsClient.ts

```diff
@@ -0,0 +1,31 @@
+import { onDocumentReady } from '@prairielearn/browser-utils';
+
+declare global {
+  interface Window {
+    histogram: any;
+    scatter: any;
+    parallel_histograms: any;
+  }
+}
+
+onDocumentReady(() => {
+  document.querySelectorAll('.histogram').forEach((histogramElement) => {
+    const data = JSON.parse(histogramElement.getAttribute('data-histogram') ?? '');
```

Comment:
I really like this implementation using classes instead of individual IDs. That said, for data attributes it's usually cleaner to use [the dataset property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset):

```suggestion
  document.querySelectorAll('.histogram').forEach((histogramElement) => {
    const data = JSON.parse(histogramElement.dataset.histogram ?? '');
```

Also a bit of a nit, but we have been moving towards using a `js-` prefix for classes that are used only for JS selection (i.e., without styling involved). So `js-histogram` instead of `histogram`.

Reply:
Added a `js-` prefix to the classes and updated to use `dataset` instead of `getAttribute('data-...`. in [00ffe80](https://github.com/PrairieLearn/PrairieLearn/pull/9809/commits/00ffe80fa4857a2dae6d76fbf63b45e21cdcecc2).